include/nbl/builtin/hlsl/bxdf/common.hlsl

```diff
@@ -854,6 +854,11 @@ enum BxDFClampMode : uint16_t
     BCM_ABS
 };
 
+template<typename T>
+using quotient_and_pdf_scalar = sampling::quotient_and_pdf<vector<T, 1>, float32_t> ;
+template<typename T>
+using quotient_and_pdf_rgb = sampling::quotient_and_pdf<vector<T, 3>, T> ;
```

Comment:
why is PDF hardcoded to `float32_t` ?

IMHO should be
```cpp
template<typename T, typename P=T>
```
and `P` be the PDF type